examples/BUILD.bazel

```diff
@@ -20,7 +20,8 @@ cc_binary(
     srcs = ["run_mock_protocol_client_main.cc"],
     deps = [
         "//src/common:system_logger",
-        "//src/common/protocol_client:chunk_server_service_client",
+        "//src/common/protocol_client:chunk_server_service_client_client",
```

Comment:
Can we rename it to something like gfs client

Reply:
You meant "chunk_server_service_client_gfs_{client,master,chunk_server}"?

Reply:
"chunk_server_service_{gfs, master_server}_client